pipeline-runner/R/qc-6-integrate_scdata.R

```diff
@@ -268,3 +287,12 @@ colorObject <- function(data) {
   }
   return(data)
 }
+
+estimate_npcs <- function(scdata, var_threshold, max_npcs) {
```

Comment:
```suggestion
get_npcs <- function(scdata, var_threshold = 0.85, max_npcs = 30) {
```
just set as default parameter values and then remove arguments when calling function